drivers/ethernet/Kconfig.e1000

```diff
@@ -13,6 +13,8 @@ menuconfig ETH_E1000
 	help
```

Comment:
Please squash this commit `drivers: eth: e1000: Add IRQ priority and flags entries` to `drivers: eth: e1000: Add driver Intel PRO/1000 Ethernet controller`